envi/qt/memcanvas.py

```diff
@@ -82,7 +81,7 @@ def renderMemory(self, va, size, rend=None):
             size += size + szdiff
 
             def callScroll(data):
-                self._scrollToVa(origva)
+                self._selectVa(origva)
```

Comment:
huh?  why call _selectVa() for "callScroll()"?  should this name be different?  or should there be a comment?